package.json

```diff
@@ -47,6 +47,7 @@
     "react-dom": "^16.5.0",
     "react-redux": "^5.0.7",
     "redux": "^4.0.0",
+    "styled-components": "^3.4.5",
```

Comment:
Let's use styled-component v4 (beta version) because it's Better, Faster and Stronger :D
https://medium.com/styled-components/announcing-styled-components-v4-better-faster-stronger-3fe1aba1a112